### PR TITLE
Enabled CI build and added json validation logic

### PR DESCRIPTION
**Pull request by ericc1103**

### CODE REVIEW

File: test/jsonValidationTest.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+                done();
+        });
+    });
+
+    it('Verifying workbook settings.json files', function(done){
+        browseDirectory(cohortPath, (error, results) => {
+            if (error) throw error;
+            results.filter(file => file.substr(-13) === 'settings.json')
+                .forEach(file => {
+                    console.log(file);
```
Comment (gardnerjr) [inline test/jsonValidationTest.js]: log needed?

File: test/jsonValidationTest.js [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+                    validateJsonString(file);
+                });
+            
+                done();
+        });
+    });
+
+    it('Verifying workbook category json files', function(done){
```
Comment (gardnerjr) [inline test/jsonValidationTest.js]: for category and settings, do you want further validation?
they have the right fields, etc.  right now this just validates that they are valid json.  so I could have a categoryResources.json that's literally `[]` and that would be valid json.

File: test/jsonValidationTest.js (hunk 1 above)
Comment (ericc1103) [reply test/jsonValidationTest.js]: Won't hurt. :-)

File: test/jsonValidationTest.js (hunk 2 above)
Comment (ericc1103) [reply test/jsonValidationTest.js]: That would be the next step. We have a lot to add.

**Pull request merged**